spec/acceptance/letsencrypt_plugin_dns_rfc2136_spec.rb

```diff
@@ -1,16 +1,6 @@
 require 'spec_helper_acceptance'
 
-describe 'letsencrypt::plugin::dns_rfc2136' do
-  supported = case fact('os.family')
-              when 'Debian'
-                # Debian 9 has it in backports, Ubuntu started shipping in Bionic
-                fact('os.release.major') != '9' && fact('os.release.major') != '16.04'
-              when 'RedHat'
-                true
-              else
-                false
-              end
-
+describe 'letsencrypt::plugin::dns_rfc2136', if: %w[Debian RedHat].include?(fact('os.family')) do
```

Comment:
I don't know. I just wanted to drop an obsolete check. Maybe we can support this two plugins for all supported operating systems and this would not be necessary.
@ekohl Do you have any inside?

Reply:
In https://github.com/voxpupuli/puppet-letsencrypt/pull/265#discussion_r768772759 I suggested the same. So I think it'd be fine to drop the whole condition. So
```suggestion
describe 'letsencrypt::plugin::dns_rfc2136' do
```

Reply:
I found packages for FreeBSD bot I could not find packages for OpenBSD. Shall I add them in a separate PR?

Reply:
I'll create a separate PR.